### PR TITLE
Adds enhancement to networking section for kuryr-kubernetes and OVN Octavia

### PR DESCRIPTION
**Pull request by stevsmit**
No BZ.

Preview: https://deploy-preview-33504--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-enhancements-kuryr-kubernetes

### CODE REVIEW

**Comment by dulek** [comment]
Looks good from Kuryr dev side!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -201,6 +201,20 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-co
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-setting-thread-count_configuring-ingress[Setting Ingress Controller thread count].
 
+[id="ocp-4-8-enhancements-kuryr-kubernetes"]
+==== Changes to default API load balancer management for Kuryr 
+
+In {product-title} 4.8 deployments on OpenStack with Kuryr-Kubernetes, the API load balancer for the `default/kubernetes` service is no longer managed by the Cluster Network Operator (CNO), but instead by the kuryr-controller itself. This means that:
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:207]: s/OpenStack/{rh-openstack-first}

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -201,6 +201,20 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-co
...
+[id="ocp-4-8-enhancements-kuryr-kubernetes"]
+==== Changes to default API load balancer management for Kuryr 
+
+In {product-title} 4.8 deployments on OpenStack with Kuryr-Kubernetes, the API load balancer for the `default/kubernetes` service is no longer managed by the Cluster Network Operator (CNO), but instead by the kuryr-controller itself. This means that: 
+
+* When upgrading to {product-title} 4.8, the `default/kubernetes` service will have downtime.
++
+[NOTE]
+====
+In deployments where no OVN Octavia is available, more downtime should be expected
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:213]: Open Virtual Network (OVN)

**Comment by ahardin-rh** [comment]
@stevsmit Just some minor comments from me. Looks like we also need a rebase now and we should be good to go. Thanks!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (stevsmit) [reply release_notes/ocp-4-8-release-notes.adoc:207]: Got it.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (stevsmit) [reply release_notes/ocp-4-8-release-notes.adoc:213]: Done.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 3]
```diff
@@ -213,6 +213,21 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-co
...
+
+* When upgrading to {product-title} 4.8, the `default/kubernetes` service will have downtime.
++
+[NOTE]
+====
+In deployments where no Open Virtual Network (OVN) Octavia is available, more downtime should be expected
+====
+
+* The `default/kubernetes` load balancer is no longer required to use the Octavia Amphora driver. Instead, OVN Octavia will be used to implement the `default/kubernetes` service if it is available in the OpenStack cloud. 
+>>>>>>> ac43e0c36... Adds entry for kuryr-kubernetes enhancement to networking section
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:230]: This >>>>>>> line was added in when there was a previous conflict, so you can remove this line

File: release_notes/ocp-4-8-release-notes.adoc [hunk 4]
```diff
@@ -213,6 +213,21 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-co
 ==== NTP servers on control plane nodes
 
 In {product-title} {product-version}, installer-provisioned clusters can configure and deploy Network Time Protocol (NTP) servers on the control plane nodes and NTP clients on worker nodes. This enables workers to retrieve the date and time from the NTP servers on the control plane nodes, even when disconnected from a routable network. You can also configure and deploy NTP servers and NTP clients after deployment.
+=======
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:216]: Same with this ======= bit

**Pull request merged**